shopify/wishlist-template/extensions/wishlists/src/components/NewWishlist.tsx

```diff
@@ -67,15 +81,9 @@ export default ({ wishlists }) => {
                   (wishlist) => wishlist.name === value
                 );
 
-                if (!passed) {
-                  setError("wishlist.name", {
-                    type: "uniqueness",
-                    message: "Name must be unique",
-                  });
-                  return "Name must be unique";
-                }
+                if (!passed) return "Name must be unique";
```

Comment:
This seems like validation that should be done on the server, not the client. We already have the field declared as unique, and should probably just handle the error instead of duplicating a check. Will also simplify this component and input a bunch